docker-compose.yml

```diff
@@ -11,14 +11,27 @@ services:
        - DATA_APP_ENDPOINT=${PROVIDER_DATA_APP_ENDPOINT}      #Data APP enpoint for consuming received data 
        - MULTIPART=${PROVIDER_MULTIPART}                      #Data APP endpoint multipart/mixed content type
        - REST_ENABLE_HTTPS=${REST_ENABLE_HTTPS}
-       - IDSCP=${PROVIDER_IDSCP}
+       - IDSCP2=${PROVIDER_IDSCP2}
        - WS_OVER_HTTPS=${PROVIDER_WS_OVER_HTTPS}
        - WS_INTERNAL=${PROVIDER_WS_INTERNAL}
        - UC_DATAAPP_URI=${PROVIDER_UC_DATAAPP_URI}
        - CLEARING_HOUSE=${CLEARING_HOUSE}
        - PUBLIC_PORT=${PROVIDER_PORT}
        - BROKER_URL=${BROKER_URL}
        - DISABLE_SSL_VALIDATION=${DISABLE_SSL_VALIDATION}
+       - CACHE_TOKEN=${CACHE_TOKEN}
+       - FETCH_TOKEN_ON_STARTUP=${FETCH_TOKEN_ON_STARTUP}
+       - SERVER_SSL_ENABLED=${SERVER_SSL_ENABLED}
+       - KEYSTORE_NAME=${KEYSTORE_NAME}
+       - KEY_PASSWORD=${KEY_PASSWORD}
+       - KEYSTORE_PASSWORD=${KEYSTORE_PASSWORD}
+       - ALIAS=${ALIAS}
+       - DAPS_KEYSTORE_NAME=${DAPS_KEYSTORE_NAME}
+       - DAPS_KEYSTORE_PASSWORD=${DAPS_KEYSTORE_PASSWORD}
+       - DAPS_KEYSTORE_ALIAS=${DAPS_KEYSTORE_ALIAS}
+       - DAPS_TRUSTORE_NAME=${DAPS_TRUSTORE_NAME}
+       - DAPS_TRUSTORE_PASSWORD=${DAPS_TRUSTORE_PASSWORD}
+       - TZ=Europe/Rome                
```

Comment:
I forgot to add this time zone to other images, can you add it also to the rest configurations?